src/dllinject/dllinject.c

```diff
@@ -2187,4 +2197,4 @@ int tup_inject_dll(
 	}
 
 	return 0;
-}
+}
```

Comment:
Can you please configure your editor to write out a newline at the end of the file? It breaks the diffs otherwise.